.circleci/config.yml

```diff
@@ -156,6 +159,6 @@ workflows:
   version: 2
   build:
     jobs:
-      - "trusty-python-3.4"
-      - "xenial-python-3.5"
-      - "bionic-python-3.6"
+      - "trusty"
+      - "xenial"
+      - "bionic"
```

Comment:
and these xenial-backend and bionic-backend?  I think maybe having these names be clear about what's special about trusty would be helpful.